### DISCUSSION TITLE
Datatable Style

### DISCUSSION THREAD

**Discussion by cheadevit** [Laravel-Backpack/community-forum#611] [Q&A (Help)] [answered]
Hi Guys,

Here is my project version:

### PHP VERSION:
PHP 8.1.16 (cli) (built: Feb 14 2023 18:34:53) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.16, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.16, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.17.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.1.1
backpack/crud: 6.1.4
backpack/generators: v4.0.2
backpack/permissionmanager: 7.0.0
backpack/pro: 2.0.10
backpack/theme-coreuiv2: 1.1.4

I face a problem after upgrading from Backpack version 5 to 6 as shown in the image below

![Screenshot 2023-08-04 at 13 52 57](https://github.com/Laravel-Backpack/community-forum/assets/52439329/c989dfcd-a70a-4dee-bd63-5800d1714b34)

**Comment by karandatwani92** [comment, marked as answer]
Hey @cheadevit, it seems like table responsiveness is disabled. 
Please check after enabling it using `CRUD::enableResponsiveTable();`

More info here https://backpackforlaravel.com/docs/6.x/crud-cheat-sheet#responsive-table
